components/MonitoringUserSetupInstructions.tsx

```diff
@@ -187,10 +187,10 @@ CREATE SCHEMA pganalyze;
 GRANT USAGE ON SCHEMA pganalyze TO pganalyze;
 GRANT USAGE ON SCHEMA public TO pganalyze;
 
-CREATE OR REPLACE FUNCTION pganalyze.get_stat_replication() RETURNS SETOF pg_stat_replication AS
+${noPgMonitor ? `CREATE OR REPLACE FUNCTION pganalyze.get_stat_replication() RETURNS SETOF pg_stat_replication AS
```

Comment:
Uhmm, didn't you remove this function from the docs already?

https://github.com/pganalyze/pganalyze-docs/pull/242/files#diff-29b4a130316538bca91876ddac10f2adc7140298e786bd334b0cc8aa71a365c0R188